agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -286,6 +286,9 @@ void backhaul_manager::platform_notify_error(int code, const std::string &error_
     message_com::send_cmdu(m_scPlatform.get(), cmdu_tx);
 }
 
+void backhaul_manager::before_select()
```

Comment:
commit message typo - "functiob" --> "function".
Can you also please elaborate a bit more on the use of the mutex in this commit message? I still don't understand it completely..

Reply:
fixed